src/Foundation/People/platform/Extenders/CustomSchemaExtender.cs

```diff
@@ -35,13 +43,23 @@ public CustomSchemaExtender()
 		private string GetLatestMvpYear(Sitecore.Data.Items.Item personItem)
 		{
 			var awards = GetAwardsTags(personItem);
-			if(awards.Any())
+			if (awards.Any())
 			{
 				return awards.First().Key.ToString();
 			}
 			return "";
 		}
 
+		private string GetLatestMvpYearList(Sitecore.Data.Items.Item personItem)
+		{
+			var years = string.Empty;
+
+			var awards = GetAwardsTags(personItem);
+			years = awards.Count().ToString();
```

Comment:
@joseamneto , please use another name for the variable year

Reply:
variable "years" removed, returning the awards directly